File: RentalManager.Infrastructure/Commands/CreateRentalAgreement.cs
namespace RentalManager.Infrastructure.Commands
{
    public class CreateRentalAgreement
    {
        public int EmployeeId { get; set; }
        public bool IsActive { get; set; }
        public int ClientId { get; set; }
        public List<int> RentalEquipmentIds { get; set; }
        public string? Comment { get; set; }
        public int Deposit { get; set; }
        public int? TransportFrom { get; set; }
        public int TransportTo { get; set; }
        public DateTime DateAdded { get; set; }
        public List<CreatePayment> Payments { get; set; }
    }
}

File: RentalManager.WebApp/src/references/codegen/API/models/CreateRentalAgreement.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { CreatePayment } from './CreatePayment';

export type CreateRentalAgreement = {
    employeeId?: number;
    isActive?: boolean;
    clientId?: number;
    rentalEquipmentIds?: Array<number> | null;
    comment?: string | null;
    deposit?: number;
    transportFrom?: number | null;
    transportTo?: number;
    dateAdded?: string;
    payments?: Array<CreatePayment> | null;
};
